internal/engine/engine_test.go

```diff
@@ -107,7 +114,7 @@ func runBenchmarks(b *testing.B, eng *Engine, testCases []test.Case) {
 	}
 }
 
-func mkEngine(tb testing.TB, enableAuditLog bool) (*Engine, context.CancelFunc) {
+func mkEngine(tb testing.TB, enableAuditLog, enableSchemaValidation bool) (*Engine, context.CancelFunc) {
```

Comment:
Let's declare a struct to hold these boolean parameters. Understanding what `mkEngine(b, true, false)` means is harder compared to something like `mkEngine(b, param{enableAuditLog: true, enableSchemaValidation: false})`